src/CommentSold/Api/Resources/AbstractResource.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommentSold\Api\Resources;
+
+abstract class AbstractResource
+{
+    public function toArray(): array
+    {
+        return (array) $this;
+    }
+
+    public function toJson(): string
+    {
+        return json_encode($this);
+    }
+}
```

Comment:
This doesn't really feel necessary. They're sort of nice helpers, but probably not worth the inheritance tax